File: core/feature39_test.go
package main

import "testing"

var (
	feature39Names      = []string{"S", "L", "Points"}
	feature39Accuracies = []string{"%.1f", "%.0f", "%.0f"}
	feature39Types      = []bool{true, false, false}
)

func Test01feature39(t *testing.T) {
	arguments := map[string]interface{}{"D": 200, "G": 1000, "I": 10, "V": 250, "T": 6}
	expected := map[string]interface{}{"S": 104.7, "L": []interface{}{628, 658}, "Points": []interface{}{0, 12, 38, 51}}
	genericTester(t, feature39(arguments), expected, feature39Names, feature39Accuracies, feature39Types)
}

File: core/feature39_test.js
var feature39Names = ["S", "L", "Points"];
var feature39Accuracies = [1, 0, 0];
var feature39Types = [true, false, false];

function test01feature39() {
  var arguments = {"D": 200, "G": 1000, "I": 10, "V": 250, "T": 6};
  var expected = {"S": 104.7, "L": [628, 658], "Points": [0, 12, 38, 51]};
  genericTester(feature39_core(arguments), expected, feature39Names, feature39Accuracies, feature39Types);
}

test01feature39();
